models/include/cloe/component/pedal_actuator.hpp

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2022 Robert Bosch GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/**
+ * \file cloe/component/pedal_actuator.hpp
+ * 
+ */
+
+#pragma once
+#ifndef CLOE_COMPONENT_PEDAL_ACTUATOR_HPP_
+#define CLOE_COMPONENT_PEDAL_ACTUATOR_HPP_
+
+#include <cloe/component/actuator.hpp>  // for Actuator
+
+namespace cloe {
+
+struct PedalRequest {
```

Comment:
while implementing this, i had similar questions as you with regards to this interface. i discussed this with some others before. what i understood was:
- with some controllers it really actuates the pedal (and it would move for the driver) (which i personally would not have expected)
- with some other controllers it does not, but currently then there is a calculation from the controller output to these values and the pedal request is then passed to the simulator.

i will try to get a clearer picture on this at the start of next week.